src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+        int startingIndex = line.indexOf("/");
+        String taskDisplay = line.substring(9, startingIndex);
+        String doBy = "(" + line.substring(startingIndex + 1) + ")";
+        char taskType = line.toUpperCase().charAt(0);
+
+        stringList[listCount] = taskDisplay;
+        Deadline deadlineTask = new Deadline(taskDisplay, taskType, doBy);
+        taskList[listCount] = deadlineTask;
+        return deadlineTask;
+    }
+
+    private static Event event(String[] stringList, Task[] taskList, int listCount, String line) {
+        int startingIndex = line.indexOf("/");
+        String taskDisplay = line.substring(6, startingIndex);
+        String doBy = "(" + line.substring(startingIndex + 1) + ")";
+        char taskType = line.toUpperCase().charAt(0);
+
+        stringList[listCount] = taskDisplay;
+        Event eventTask = new Event(taskDisplay, taskType, doBy);
+        taskList[listCount] = eventTask;
+        return eventTask;
+    }
+
+    private static void printTask(Task task, int listCount) {
+        System.out.println("Got it. I've added this task:");
+        System.out.println(task);
+        System.out.println("Now you have " + listCount + " tasks in the list.");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+
+        String[] stringList = new String[100];
+        Task[] taskList = new Task[100];
+        int listCount = 0;
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        while (!line.equals("bye")) {
+            if (line.startsWith("done")) {
+                done(line, taskList);
+
+            } else if (line.equals("list")) {
+                list(taskList, listCount);
+
+            } else if (line.startsWith("todo")) {
+                Todo todoTask = todo(stringList, taskList, listCount, line);
+                listCount++;
+                printTask(todoTask, listCount);
+
+            } else if (line.startsWith("deadline")) {
+                Deadline deadlineTask = deadline(stringList, taskList, listCount, line);
+                listCount++;
+                printTask(deadlineTask, listCount);
+
+            } else if (line.startsWith("event")) {
+                Event eventTask = event(stringList, taskList, listCount, line);
+                listCount++;
+                printTask(eventTask, listCount);
+
+            } else {
+                System.out.println("Invalid command");
+            }
+            line = in.nextLine();
+        }
+        System.out.println("Bye. Hope to see you again soon!");
```

Comment:
Consider defining your messages as static finals on the top of your code so you can edit easier in the future :)